packages/react/src/MinimalFooter/MinimalFooter.tsx

```diff
@@ -0,0 +1,272 @@
+                </Text>
+                <Stack
+                  padding="none"
+                  gap="condensed"
+                  justifyContent={{
+                    narrow: 'center',
+                    regular: 'flex-end'
+                  }}
+                  direction={{
+                    narrow: 'vertical',
+                    regular: 'horizontal'
+                  }}
+                  className={styles['Footer__links']}
+                >
+                  <>{LinkChildren}</>
+                </Stack>
+              </Stack>
+            </div>
+          </div>
+        </section>
+      </footer>
+    </ThemeProvider>
+  )
+}
+
+type FootnoteProps = BaseProps<HTMLElement>
+
+function Footnotes({children, className}: PropsWithChildren<FootnoteProps>) {
+  const styledChildren = React.Children.map(children, child => {
+    // if not valid element
+    if (!React.isValidElement(child)) {
+      return child
+    }
+
+    if (child.type && child.type === Text) {
+      return React.cloneElement(child as React.ReactElement, {
+        as: 'p',
+        variant: 'muted',
+        size: 200,
+        className: clsx(styles['Footer__terms-item'], child.props.className),
+        ...child.props // allow overrides for escape hatch
+      })
+    }
+  })
+
+  return (
+    <section className={styles.Footer__container}>
+      <div className={clsx(styles.Footer__terms, className)}>{styledChildren}</div>
+    </section>
+  )
+}
+
+type SocialLogomarksProps = {
+  socialLinks?: SocialLinks[] | false
+  logoHref?: string
+}
+
+function SocialLogomarks({socialLinks, logoHref}: SocialLogomarksProps) {
+  const socialLinkData = [
+    {
+      name: 'twitter',
+      fullName: 'Twitter',
+      url: 'https://twitter.com/github',
+      icon: 'https://github.githubassets.com/images/modules/site/icons/footer/twitter.svg',
+      iconWidth: 22,
+      iconHeight: 18
+    },
+    {
+      name: 'github',
+      fullName: 'GitHub',
+      url: 'https://github.com',
+      icon: 'https://github.githubassets.com/images/modules/site/icons/footer/github-mark.svg',
+      iconWidth: 20,
+      iconHeight: 20
+    },
+    {
+      name: 'linkedin',
+      fullName: 'LinkedIn',
+      url: 'https://www.linkedin.com/company/github',
+      icon: 'https://github.githubassets.com/images/modules/site/icons/footer/linkedin.svg',
+      iconWidth: 19,
+      iconHeight: 18
+    },
+    {
+      name: 'youtube',
+      fullName: 'YouTube',
+      url: 'https://youtube.com',
+      icon: 'https://github.githubassets.com/images/modules/site/icons/footer/youtube.svg',
+      iconWidth: 23,
+      iconHeight: 16
+    },
+    {
+      name: 'facebook',
+      fullName: 'Facebook',
+      url: 'https://facebook.com',
+      icon: 'https://github.githubassets.com/images/modules/site/icons/footer/facebook.svg',
+      iconWidth: 18,
+      iconHeight: 18
+    }
```

Comment:
On the large footer, we have links for [Twitch](https://www.twitch.tv/github) and [TikTok](https://www.tiktok.com/@github). Shall we include those too?

Reply:
Added! thanks